fiftyone/utils/open_clip.py

```diff
@@ -143,7 +143,7 @@ def _predict_all(self, imgs):
         frame_size = (width, height)
 
         if self._using_gpu:
-            imgs = imgs.cuda()
+            imgs = imgs.to(self.device)
```

Comment:
@jacobsela coderabbit raises an interesting point here. Does `torch.amp.autocast("cuda")` need to be updated?

Reply:
Yeah this is valid and will cause problems if not handled. It's in my todo for this week to more thoroughly review the code before moving further with this PR because this message makes me think that there are probably more places I haven't noticed that make hardware assumptions.